make/tools.mk

```diff
@@ -25,14 +25,11 @@ endif
 QT_VERSION := 5.6.1
 QT_SDK_DIR := $(TOOLS_DIR)/Qt$(QT_VERSION)
 
-ifndef IGNORE_MISSING_TOOLCHAIN
-  # On any platform except windows
-  ifndef WINDOWS
-    # Check for a current QT SDK dir, abort without
-    ifeq ($(wildcard $(QT_SDK_DIR)/*),)
-      ifeq (,$(findstring _install,$(MAKECMDGOALS)))
-        $(error "QT SDK not found, please run `make qt_sdk_install`")
-      endif
+ifndef WINDOWS
+# Check for a current QT SDK dir, abort without
+  ifeq ($(wildcard $(QT_SDK_DIR)/*),)
+    ifeq (,$(findstring _install,$(MAKECMDGOALS)))
```

Comment:
The main purpose of this is so you can build sim-- if you're doing that, you deserve the warning, but it's OK if it proceeds/succeeds IMO.